bug: Cannot resolve package primitives

```
../../packages/client/src/providers/app.provider.tsx:2:0
Module not found: Can't resolve '@universal-labs/primitives'
  1 | import { View } from 'react-native';
> 2 | import { H1 } from '@universal-labs/primitives';
  3 | 
  4 | interface IAppProviderProps {
  5 |   children: React.ReactNode;

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
../../packages/client/src/index.ts
./src/pages/_app.tsx
```

--- a/.changeset/bright-chefs-sin.md
+++ b/.changeset/bright-chefs-sin.md
@@ -1,0 +1,10 @@
+---
+'@universal-labs/core': patch
+'@universal-labs/hooks': patch
+'@universal-labs/primitives': patch
+'@universal-labs/stylesheets': patch
+'@universal-labs/styled': patch
+'@universal-labs/tailwind-ui': patch
+---
+
+publish files fix

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -45,14 +45,16 @@ jobs:
       - name: Install Dependencies
         run: yarn --immutable
 
-      - name: Create Release Pull Request or Publish to npm
-        id: changesets
-        uses: changesets/action@v1
-        with:
-          publish: yarn release
-          commit: version-packages
-          title: Version Packages
-          createGithubReleases: true
+      - name: Create Release and Publish to npm
+        run: |
+          yarn changeset version
+          yarn workspaces foreach --no-private --verbose --from '@universal-labs/*' npm publish --access public
+          yarn changeset tag
+        # with:
+        #   publish: yarn release
+        #   commit: version-packages
+        #   title: Version Packages
+        #   createGithubReleases: true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}

